README.md

```diff
@@ -133,7 +133,75 @@ For a module to act as the root module the only requirements are:
 
 ### Making an API call
 
-### Adding a Route
+### Routing and Navigation
+
+One App contains a single route `<ModuleRoute moduleName={root-module-name}/>` which loads the
+root-module configured by `root-module-name`. This is what makes the root module the entry
+point to your application. As One App does not set a path for the root module the root module
+itself is required to have at least a single route with a path.
+
+```js
+import { Route } from '@americanexpress/one-app-router';
+
+export function RootModule({ children, config }) {
+  return <h1>Hello World!</h1>;
+}
+RootModule.childRoutes = [
+  <Route path="/" />,
```

Comment:
is path the only required prop?

Reply:
for the router to match and load the child module, yes